code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -70,8 +70,10 @@
 			return
 
 	if (M.can_inject(user, 1))
-		user << "<span class='notice'>You inject [M] with the injector.</span>"
-		M << "<span class='notice'>You feel a tiny prick!</span>"
+		visible_message("<span class='notice'>[user] starts to inject [M] with their hypospray!</span>", "<span class='notice'>You start to inject [M] with your hypospray!</span>")
+		if(do_mob(user, M, 3 SECONDS))

```

Comment:
I think you should just keep the borg hypospray instant and instead remove soporific from the thing.